### ISSUE TITLE
Windows: Always returns exit code 0 even in failure cases

### ISSUE THREAD

**Issue by dchengTSC** [ssciwr/clang-format-wheel#23]
I am trying to roll out the clang-format pre-commit hook to my mac+windows dev team, which uses clang-format-wheel. Everything works great on the mac side, but there are a few issues on the Windows side.

A known issue is that the [Windows side is missing dependencies](https://github.com/ssciwr/clang-format-wheel/issues/22). I appreciate that is it being worked on.

An additional issue, however, is that clang-format-wheel isn't correctly bubbling up the error case. It seems that if DLLs are missing, clang-format should return a non-zero exit code. I opened up an [issue in mirrors-clang-format](https://github.com/pre-commit/mirrors-clang-format/issues/11) and was redirected to post the issue here.

A few other observations I made:
* When running clang-format via pre-commit through cmd.exe, I get a ton of the "missing DLL" dialog popups as seen in https://github.com/ssciwr/clang-format-wheel/issues/22. I expect pre-commit to fail (based on the clang-format exit code), but it succeeds.
* When running clang-format via pre-commit through Powershell, I don't get any of the "missing DLL" dialog popups, but clang-format definitely did not run because the corrections weren't made. pre-commit continues to show success, which indicates that the error code wasn't bubbled up.

**Comment by dokempf** [comment]
The solution for the erroneous passes is to switch from `os.evecv` to `subprocess.call` like e.g. done for the Python CMake distribution: https://github.com/scikit-build/cmake-python-distributions/blob/master/src/cmake/__init__.py#L43

**Comment by dokempf** [comment]
@dchengTSC Could you try whether installing [this version from TestPyPI](https://test.pypi.org/project/clang-format/13.0.1) solves this issue and #22 ?

**Comment by dchengTSC** [comment]
That version of TestPyPI fixes both problems. Thank you!

To verify this issue, I tested this in powershell:

```
clang-format -i --style=file -Werror --dry-run IncorrectlyFormattedFile.cpp
echo $? #output was False. Previously was True
```

In cmd:

```
clang-format -i --style=file -Werror --dry-run IncorrectlyFormattedFile.cpp
echo %errorlevel%
```
The output here was 1

In the cmd case, if I removed `-Werror`, it correctly formatted the file without popping up those dialogs, so that means #22 is fixed as well.

**Comment by dokempf** [comment]
Thanks for the fast response. The fixes are merged and v13.0.1 is on its way. I will add releases for the 10-12 series tomorrow during the day.
